testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -935,6 +935,8 @@ impl BurnchainController for BitcoinRegtestController {
             BlockstackOperationType::UserBurnSupport(payload) => {
                 self.build_user_burn_support_tx(payload, op_signer, attempt)
             }
+            BlockstackOperationType::PreStackStx(_payload) => unimplemented!(),
+            BlockstackOperationType::StackStx(_payload) => unimplemented!(),
```

Comment:
Maybe this is a separate PR, but is there any miner support planned for on-BTC stacking operations?  If not, can you open an issue so we can test out stacking on the testnet?

Reply:
I don't plan on implementing it, and I don't see it as a vital element of this repo -- it seems like something that is strictly within the domain of client tooling.